regression/abstraction/abst-funcs.c

```diff
@@ -0,0 +1,714 @@
+#include <assert.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+// Some helper functions.
+
```

Comment:
Why not just have `size_t nondet_size_t{)` in the header and use that CPROVER doesn't issue 'missing function' warnings for functions whose names start with `nondet`.

Reply:
Not sure I understand this. There is no header for abst-funcs.c. It's there for rr-abstraction to choose abstraction functions from.

Reply:
I think the problem might be that there is no explanation of how this works.  My point is that if you are manually modelling things, simply having:

```
size_t nondet_size_t(void);
```

is sufficient and gives you the same behaviour.